docs/frontend.md

```diff
@@ -1,5 +1,44 @@
 # Front-end Architecture
 
+While the Login.gov application is architected as a Ruby on Rails project and most of its pages are
```

Comment:
This looks helpful.  Would it be worth including anything about how asset files are fingerprinted?  I ask because I know that migrations are responsible for compiling assets, but I am still a little foggy on how assets are fingerprinted.

Reply:
> This looks helpful. Would it be worth including anything about how asset files are fingerprinted?

I think that makes sense, though it might be better to have a section dedicated to breaking down the build pipeline, since there's a lot more that goes into that than would make sense to include in a high-level overview like this.

Reply:
> Though it might be better to have a section dedicated to breaking down the build pipeline, since there's a lot more that goes into that than would make sense to include in a high-level overview like this.
Good point. 👍🏻

Reply:
I'll plan to add a "Build Process" subheading under "JavaScript" in a follow-up pull request.